tests/container_test.go

```diff
@@ -53,11 +57,12 @@ func newContainerInvoker(t *testing.T, autohashes bool) (*neotest.ContractInvoke
 		container.RegistrationFeeKey, int64(containerFee),
 		container.AliasFeeKey, int64(containerAliasFee))
 	deployBalanceContract(t, e, ctrNetmap.Hash, ctrContainer.Hash)
-	var addrNetmap, addrBalance, addrNNS *util.Uint160
 	if !autohashes {
-		addrNetmap, addrBalance, addrNNS = &ctrNetmap.Hash, &ctrBalance.Hash, &nnsHash
+		deployContainerContract(t, e, &ctrNetmap.Hash, &ctrBalance.Hash, &nnsHash)
+	} else {
+		_ = deployNeoFSIDContract(t, e) // Needed in this case to autoresolve.
```

Comment:
easy to forget, so lets do inside `deployContainerContractInt` or at least write comment

Reply:
`deployContainerContractInt` deploys `container` only, it doesn't care about dependencies. Dependencies are handled here.

Reply:
this doesnt change my request

what `Int` stays for in `deployContainerContractInt`? Internal? Integer?

Reply:
Internal.